File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x< 0:
            return False 
        original = x
        sum =0
        while x >0 :
            num = x%10
            sum = sum *10 + num 
            x = x//10
        return original == sum 

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        // Negative numbers are not palindromes
        if(x <0){
            return false;
        }
        int original = x;
        int sum =0;
        while (x!=0){
            int num = x%10;
            sum = sum *10 +num;
            x = x/10;
        }
        return original == sum;
    }
}